frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -375,14 +383,25 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
               onChange={React.useCallback((diskBus) => setBus(DiskBus.fromString(diskBus)), [
                 setBus,
               ])}
+              isValid={!isValidationError(busValidation)}
               value={asFormSelectValue(bus)}
               id={asId('interface')}
               isDisabled={inProgress}
             >
               <FormSelectPlaceholderOption isDisabled placeholder="--- Select Interface ---" />
-              {DiskBus.getAll().map((b) => (
-                <FormSelectOption key={b.getValue()} value={b.getValue()} label={b.toString()} />
-              ))}
+              {!(allowedBusses || new Set(DiskBus.getAll())).has(bus) && (
+                <FormSelectOption
+                  isDisabled
+                  key={bus.getValue()}
+                  value={bus.getValue()}
+                  label={`${bus.toString()} (Invalid)`}
+                />
+              )}
+              {DiskBus.getAll().map((b) =>
```

Comment:
can we do the same thing here and just map the thing without any conditions?

```suggestion
              {(allowedBusses || new Set(DiskBus.getAll())).map((b) =>
```

we probably should check if allowedBusses are null in the begiining and assing new Set(DiskBus.getAll()) so we can use it everywhere without a fear